js/index.html

```diff
@@ -43,28 +43,100 @@ <h3> </h3>
+        // you might already have conversations under `application.conversations[]` to get,
+        // or join (if application.conversations[xxx].me.state === &quot;joined&quot; || &quot;invited&quot; respectively)
+
+        application.newConversation().then( //not providing a name, the service will assign a random one for you
+            function(conversation) {
+                // join the created conversation
+                // you can also find your user_id under rtc.application.me (as application === rtc.application)
+                conversation.join().then(
+                    function(member) {
+                        console.log(&quot;Joined as &quot; + member.user.name); //member.id is your member's id
+                    });
+
+                // listen for incoming text messages
+                conversation.on(&quot;text&quot;, function(sender, textEvent){
+                        // manage seen indication
+                        // if (rtc.isVisible) { textEvent.seen(); }
+                        console.log(&quot;message received:&quot;, textEvent, sender);
+                });
+
+                // send a text event to the conversation
+                conversation.sendText(&quot;Hi Nexmo&quot;).then(
+                    function(){
+                        console.log('message was sent');
+                    }).catch(function(error){
+                        console.log('error sending the message', error);
+                });
+            }).catch(function(error) {
+            console.log(error);
+        });
+    });</code></pre><p>The main objects here are <code>application</code> and <code>conversation</code> you can use them to extend the use case.
+E.g. by inviting a second user to a conversation that you are a member (have joined).
+The second instance should see the conversation listed, and the member(or me).state as &quot;invited&quot;
+When both instances have attached listener to &quot;text&quot; events of a conversation they have both joined,
+you should be able to send/receive text events in that conversation.</p>
+<h2>Build</h2><p>If you want to contribute:</p>
+<h3>Install SDK</h3><p>clone this repo and enter in it's directory</p>
+<pre class="prettyprint source lang-shell"><code>$ npm install</code></pre><h3>build</h3><pre class="prettyprint source lang-shell"><code>$ npm start</code></pre><p>it will build interracially your code.</p>
+<p>or</p>
+<pre class="prettyprint source lang-shell"><code>$ npm run build</code></pre><h3>generate docs</h3><pre class="prettyprint source lang-shell"><code>$ npm run docs</code></pre><p>it will generate your code in <code>dist/conversationClient</code></p>
+<h3>Run the tests</h3><p>you can run the test with</p>
+<pre class="prettyprint source lang-shell"><code>$ npm run test</code></pre><p><a href="https://www.nexmo.com"><img src="https://www.nexmo.com/wp-content/themes/nexmo/resources/img/nexmo_logo_157x43_fullcolor.png" alt="NEXMO"></a></p></article>
```

Comment:
I'm using a different Readme file to generate the docs. Can you please use this as well? https://github.com/AlexLakatos/conversation-js-sdk/blob/docs/README.md